.github/workflows/golang_linting.yml

```diff
@@ -0,0 +1,26 @@
+---
+name: golang_linting
+
+on:
+  workflow_call:
+
+jobs:
+  linting:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Setup Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+          check-latest: true
+
+      - name: Get needed modules
+        run: go mod tidy
+
```

Comment:
Maybe this is not needed for linting

Reply:
In my local tests this was needed to force the install of the modules. If not the linting job fails.